### PR TITLE
sdk/state, specifications: use payment instead of update where applicable

### PR DESCRIPTION
**Pull request by acharb**
**WHAT**
use `payment` to refer to steps where we're updating the channel state for a payment

**WHY**
currently we're using `update` to refer to payment steps. Technically to make a payment you are updating the channel state. However this collides with the `update` steps for updating the channel with non payment data like the observation period or depositing new funds. All of which are also updating the channel state.

No code changes, just file name, diagrams, and SEP changes

closes https://github.com/stellar/experimental-payment-channels/issues/85

### CODE REVIEW

File: specifications/sep-payment-channel-transactions.md [hunk 1]
```diff
@@ -229,24 +229,25 @@ multisig accounts. F has source account E, and sequence number set to s.
...
-- D_i, see [Update](#Update) process.
+- D_i, see [Payment](#Payment) process.
 
-#### Update
+#### Payment
 
-Participants use the update process to agree, and re-agree, on a new final state
+Participants use the payment process to agree, and re-agree, on a new final state
 for the channel. Participants will agree on a new final state when making
 payments to one another within the channel.
```
Comment (leighmcculloch) [inline specifications/sep-payment-channel-transactions.md:240]: Needs wrapping:
```suggestion
Participants use the payment process to agree, and re-agree, on a new final
state for the channel. Participants will agree on a new final state when making
payments to one another within the channel.
```

File: specifications/sep-payment-channel-transactions.md [hunk 2]
```diff
@@ -229,24 +229,25 @@ multisig accounts. F has source account E, and sequence number set to s.
   The `CHANGE_TRUST` operations configure the trustlines with the maximum limit,
   which is the maximum value of an `int64`, `0x7FFFFFFFFFFFFFFF`.
 
-- C_i, see [Update](#Update) process.
+- C_i, see [Payment](#Payment) process. The setup C_i will be a Payment of 0.
```
Comment (acharb) [inline specifications/sep-payment-channel-transactions.md:232]: imo this is clearer now. However I'm interested if you think this actually is less clear

**Review by leighmcculloch — APPROVED** [review]
Looks great! Can you wrap the text blocks changed so they stay under 80 for consistency with the rest of the doc? I opened suggestions on the two spots in case it is inconvenient for you to do that with your editor.

File: specifications/sep-payment-channel-transactions.md (hunk 2 above)
Comment (leighmcculloch) [reply specifications/sep-payment-channel-transactions.md:232]: Hmm, technically there won't be a payment. We'll omit the payment operation if there is no payment to make.

Comment (leighmcculloch) [reply specifications/sep-payment-channel-transactions.md:232]:
```suggestion
- C_i, with no payment, see [Payment](#Payment) process.
```

Comment (leighmcculloch) [reply specifications/sep-payment-channel-transactions.md]: I realize looking at the text further down this isn't particularly obvious, so I made it clearer in https://github.com/stellar/experimental-payment-channels/pull/167.

**Review by leighmcculloch — APPROVED** [review]
:shipit:

File: specifications/sep-payment-channel-transactions.md [hunk 3]
```diff
@@ -229,24 +229,25 @@ multisig accounts. F has source account E, and sequence number set to s.
...
-agreeing on a new declaration and aclosing transaction that supersedes all
-previous declaration and closing transaction and that will disburse $8 to I and
+to R. If I makes a payment of $2 to R, this payment process will involve I and R
+agreeing on a new declaration and closing transaction that supersedes all
+previous declaration and closing transactions and that will disburse $8 to I and
 $22 to R.
 
-To update the payment channel state, the participants:
+To make a payment, participants agree on a new payment channel state. The participant
+steps for which are:
```
[the remark this answers is not in the document]
Comment (acharb) [reply specifications/sep-payment-channel-transactions.md]: found some other spots as well:
https://github.com/stellar/experimental-payment-channels/pull/166/commits/ac8f4445293095fd065a55f58dbe1a630c94b858

**Comment by leighmcculloch** [comment]
This also closed #18.

**Pull request merged**